TODO.md

```diff
@@ -1,47 +1,35 @@
 ## High level TODO *(rough priority/dependency order)*</sub>
 
-- Generate all output rows that are in ground truth (~40% complete currently)
-  - I estimate this task is ~80% of the remaining work
+- Generate all output rows that are in ground truth ~~(~40% complete currently)~~
   - See CI for latest status
-  - See https://github.com/microsoft/times-excel-reader/wiki/State-of-Progress
+  - ~~See https://github.com/microsoft/times-excel-reader/wiki/State-of-Progress~~
 - Decide how we should manage work list and assignments - TODO file, Wiki, Issues, Milestones
 - Write an overview of how code currently works to help get new people started
 - Go over old TODO lists below and work out what is still relevant
-- Code formatting
+- ~~Code formatting~~
 - Integrate type checker into CI (mypy?)
-- Decide at what point to reach out to TIMES community and announce project (and ask for help?)
-- Figure out how to handle columns that start with * but are not comments (manual isn't clear on this)
-- Decide what to do about ~TFM_FILL. These tables look like the Excel files are meant to be updated by the tool
+- ~~Decide at what point to reach out to TIMES community and announce project (and ask for help?)~~
+- ~~Figure out how to handle columns that start with * but are not comments (manual isn't clear on this)~~
+- ~~Decide what to do about ~TFM_FILL. These tables look like the Excel files are meant to be updated by the tool~~
```

Comment:
I forget, what did we decide to do about this? Not modify the Excel files, but modify the in-memory tables so that the output reflects the updates?